content/twim-guide.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Writing a good \"This week in Matrix\" entry"
++++
+
+This week in Matrix, which usually gets called TWIM, is a concept of posting
+small but precise project updates. The blogpost gets published every friday on
+the [matrix.org blog](/blog/twim) and aims to allow the community to show off their
+progress in the Matrix ecosystem.
+
+## Prerequisites
+
+With that there are some guidelines you should be aware of before publishing your
+update to TWIM:
+
+Like all things on matrix.org your Post has to follow and comply with the
+[Matrix.org Code of Conduct](/legal/code-of-conduct). Additionally the Authors
+hold the right to not accept entries of software which seem potetially dangerous
+or unethical. These things also get cleared with T&S as required to get an unbiased
+second opinion.
+
+## Publishing
+
+For publishing to TWIM all you need is a Matrix account and participate in the
+[#thisweekinmatrix:matrix.org](https://matrix.to/#/#thisweekinmatrix:matrix.org)
+matrix room.
+
+After joining you should start your post with a mention of the user "TWIM".
+Easiest to do this is to use your client's mention completion feature.
+After that mention everything contained in the message will be part of the entry.
+If you want to post more than one project please split each into an individual entry.
+
+Images and other media are supported as well. To add them you should react with
+`⭕` on your media file. Please make sure to do this after your entry so it gets added
+to the correct entry and not on another project of yours.
+
+## How to make it a good entry
+
+Making a good entry can have some challenges. Below are some recommendations
+which work well in TWIM and which help the authors to lower their workload when
+publishing the post.
+
+### Markdown
+
+The bot expected markdown as part of the plaintext event body. Please use a client which
+does send the plaintext as markdown. While we currently accept other formats as well
+it will add manual work for the post by a human.
+
+### Do not treat it as a Changelog
```

Comment:
Thank you for adding this. <3 TWIM has recently become a dumping ground of unfiltered changelogs.

Reply:
Yes. We mostly didnt react on that since we had no guidelines. So this helps with that hopefully.